webroot/adm/administrators.html

```diff
@@ -110,18 +110,40 @@ <h3>Output</h3>
         });
 
         $('#doDisable').on('click', function () {
-            var email = encodeURIComponent($('#email').val());
-            var url = '/api/admin/disable?name=' + email;
+            var email = $('#email').val();
+            var encodedEmail = encodeURIComponent(email);
+            var url = '/api/admin/disable?name=' + encodedEmail;
+
+            if (!validateEmail(encodedEmail)) return;
+
+            const confirmationMessage = `Disabling this admin key will prevent the associated account from being to interact with the admin service.\n\nAre you sure you want to disable ${email}?`;
+
+            if (!confirm(confirmationMessage)) return;
```

Comment:
Just curious where is this `confirm()` function defined?

Reply:
good question - it comes for free with Javascript Window. See:
-  https://www.w3schools.com/jsref/obj_window.asp
- https://www.geeksforgeeks.org/javascript-window-confirm-method/

Reply:
I wasn't familiar with it before this ticket, but we are already using it elsewhere in the project